DEVELOP.md

```diff
@@ -0,0 +1,37 @@
+# Developing for iDaVIE
+## Development environment
+Before developing for iDaVIE, we strongly recommend that, in addition to the steps listed in the [Building from source](https://github.com/idia-astro/iDaVIE/blob/main/BUILD.md#prerequisites) document, the following steps are followed:
+### Install a code editor
+Unity will require the installation of [Visual Studio](https://visualstudio.microsoft.com/) as part of its installation, since that provides the compiler and libraries for the C# aspects of Unity. While it can serve as an IDE, we do not recommend it due to the heavy performance load it demands when in use. Instead, we recommend the following IDEs:
```

Comment:
will *recommend* the installation...

Reply:
How else are the C# compiler and .NET libraries installed? I don't recall seeing an option to not install Visual Studio on the first install of Unity.